docker-compose.yml

```diff
@@ -216,5 +218,7 @@ volumes:
         external: true
     data-search:
         external: true
+    data-search-logs:
```

Comment:
this should be seen by https://github.com/libero/sample-configuration/blob/master/.docker/.external-volumes.rc and created automatically